bot.mjs

```diff
@@ -36,12 +37,13 @@ client.on('message', async (message) => {
   }
 
   // summon Bobby B
+  let bobbyb;
   if (message.content.toLowerCase().includes('bobby b')) {
-    await summonBobbyB(message);
+    bobbyb = await summonBobbyB(message);
   }
 
   // summon Dog
-  await summonDog(message);
+  const dog = await summonDog(message);
 
   // check for prefix
   if (!message.content.startsWith(PREFIX)) return;
```

Comment:
If a message is sent that does not invoke a command with the prefix, it will just return and do nothing. So in the case of your change on line 135, I don't understand why are only displaying Command Not Found when the user has not done something with bobby b or the Hound. Wouldn't this open up the case for the user to make up some fake command such as `!ti sword freefolk bobby b` which is a completely invalid command, and have it not say Command not found?

Reply:
Very true. I'm going to remove that check. I think maybe bobby b and the hound shouldn't react to messages with the prefix at all.